src/cyclonedx/Commands/MergeCommand.cs

```diff
@@ -61,7 +65,55 @@ public static async Task<int> Merge(MergeCommandOptions options)
                 return (int)ExitCode.ParameterValidationError;
             }
 
-            var inputBoms = await InputBoms(options.InputFiles, options.InputFormat, outputToConsole).ConfigureAwait(false);
+            List<string> InputFiles;

```

Comment:
This should all be refactored into a separate function instead of adding 50 more lines to this already long function.  Having a separate function to determine the input files also makes that functionality more easily testable.

Reply:
Good point, I'll give it a shot. No promises though - these are my first steps with C# and it was a bumpy ride to get this far that it actually works ;)

Reply:
VScode did make it easy, done ":)